docs/source/examples/beam_output.txt

```diff
@@ -8,8 +8,8 @@ Free Variables
           dx : 1.2                                             [m]   Length of an element
      \vec{M} : [ 1.98e+03  1.27e+03  713       317      ... ]  [N*m] Internal moment     
      \vec{V} : [ 660       528       396       264      ... ]  [N]   Internal shear      
-\vec{\theta} : [ 0         0.177     0.285     0.341    ... ]        Slope               
-     \vec{w} : [ 0         0.106     0.384     0.759    ... ]  [m]   Displacement        
+\vec{\theta} : [  -        0.177     0.285     0.341    ... ]        Slope               
+     \vec{w} : [  -        0.106     0.384     0.759    ... ]  [m]   Displacement        
```

Comment:
I can explain this one tomorrow in person! It changed at first because of the unexpected side effects caused by not copying...